pkg/windows/windows.go

```diff
@@ -395,7 +396,18 @@ func (vm *windows) RebootAndReinitialize() error {
 	if _, err := vm.Run("Restart-Computer -Force", true); err != nil {
 		return fmt.Errorf("error rebooting the Windows VM: %w", err)
 	}
-	// Reinitialize the SSH connection after the VM reboot
+
+	// Wait for instance to be unreachable via SSH, implies reboot is underway
+	err := wait.PollUntilContextTimeout(context.TODO(), retry.WindowsAPIInterval, retry.ResourceChangeTimeout, true,
+		func(ctx context.Context) (bool, error) {
+			_, err := vm.Run("Get-Help", true)
+			return (err != nil), nil
+		})
+	if err != nil {
+		return fmt.Errorf("instance reboot failed to start: %w", err)
+	}
```

Comment:
This is a better approach. I suggest making this a helper for readability i.e. `vm.isUnreachable()`.